Mathlib/Analysis/Convex/StronglyConvex.lean

```diff
@@ -0,0 +1,149 @@
+/-
+Copyright (c) 2023 Chenyi Li. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chenyi Li, Ziyu Wang
+-/
+import Mathlib.Analysis.InnerProductSpace.PiL2
+
+/-!
+# Stongly convex functions from ℝⁿ to ℝ
+
+In this file, we define the strongly convex functions and prove the basic properties
+of a strongly convex function.
+We prove the equivalent definitions for a strongly convex functions.
+We prove the tighter bound for a strongly bound comparing with a general convex function.
+We prove that if a strongly convex function admits a feasible global minima,
+then the minima is unique.
+
+## TODO
+
+Add derivative information for a strongly convex functions, and prove some relevant properties.
+Connect with L_smooth property and prove more theorems.
+-/
+
+variable {n : Type _} [Fintype n]  {E : Type*} [NormedAddCommGroup E] [InnerProductSpace ℝ E]
+
+/-- Function f (x) is m - strongly convex iff
+  f (x) - m * ‖x‖ ^ 2 / 2 is a convex function -/
+def StronglyConvexOn (s : Set E) (m : ℝ) (f : E → ℝ) : Prop :=
+ m > 0 ∧ ConvexOn ℝ s (fun x ↦ f x - m / 2 * ‖x‖ ^ 2)
+
+variable (s : Set E) {f : E → ℝ}
+
+lemma equiv_lemma {x y : E} {o m: ℝ} (h₁ : o > 0) (h₂ : o < 1):
+o * (f x - m / 2 * ‖x‖ ^ 2) + (1 - o) * (f y - m / 2 * ‖y‖ ^ 2)
+  + m / 2 * ‖o • x + (1 - o) • y‖ ^ 2 =
+o * f x + (1 - o) * f y - m / 2 * o * (1 - o) * ‖x - y‖ ^ 2 :=
```

Comment:
Could you check that your code follows our [style recommandations](https://leanprover-community.github.io/contribute/style.html). In this case, the statement of the lemma should be indented twice. Thanks!

Reply:
Thank you for your advice and I will fix it and push on github again